giraffe/package.json

```diff
@@ -105,5 +105,8 @@
   "peerDependencies": {
     "react": "^16.8.0",
     "react-dom": "^16.8.0"
+  },
+  "dependencies": {
+    "merge-images": "^2.0.0"
```

Comment:
Should this be in giraffe/giraffe/package.json?

Reply:
yes

Reply:
@TCL735 good check but this it actually is the giraffe/giraffe/package.json file. from the root we're in now there's:

```
package.json
   giraffe/package.json
   stories/package.json
```